### PR TITLE
Event Organizer

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
rebase after #80

### CODE REVIEW

**Comment by nikhilgupta1211** [comment]
@ChrisBr @bgeuken the abilities are properly working now please have a look at this :)

**Comment by nikhilgupta1211** [comment]
event_organizer#index
image

event_organizer#new
image

**Comment by bgeuken** [comment]
UI looks good to me.

File: spec/features/event_email.spec.rb [hunk 1]
```diff
@@ -3,6 +3,26 @@
 feature 'Email Events', '' do
   fixtures :all
 
+  scenario 'When logged in as a requester' do
```
Comment (ChrisBr) [inline spec/features/event_email.spec.rb]: Filename is wrong: s/spec/features/event_email.spec.rb/spec/features/event_email_spec.rb/

File: app/controllers/event_organizers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class EventOrganizersController < InheritedResources::Base
+  autocomplete :user, :email
+  belongs_to :event
+  actions :all, except: [:show, :edit, :update]
+
```
Comment (bgeuken) [inline app/controllers/event_organizers_controller.rb]: @nikhilgupta1211 Are we doing any authorization for this controller?

Comment (nikhilgupta1211) [inline app/controllers/event_organizers_controller.rb]: yes https://github.com/openSUSE/travel-support-program/pull/83/files#diff-e9c04d21b8c4d777e4231c4c6ce5f8cbR152

File: spec/features/event_email.spec.rb (hunk 1 above)
Comment (nikhilgupta1211) [inline spec/features/event_email.spec.rb]: Fixed in #85

File: spec/models/ability_for_administrative_spec.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@
 require 'cancan/matchers'
 # require 'ruby-debug'
 
+# rubocop:disable BlockLength
```
Comment (bgeuken) [inline spec/models/ability_for_administrative_spec.rb]: Please squash this commit into the one that introduced the rubocop failure

Comment (nikhilgupta1211) [inline spec/models/ability_for_administrative_spec.rb]: yes ok :)

**Pull request merged**